app/assets/javascripts/float-kit/addon/components/d-float-body.gjs

```diff
@@ -52,7 +52,7 @@ export default class DFloatBody extends Component {
   <template>
     <DFloatPortal
       @inline={{@inline}}
-      @portalOutletElement={{@instance.portalOutletElement}}
+      @portalOutletElement={{(@instance.portalOutletElement)}}
```

Comment:
Im also not sure about this one, but that feels more correct, having a property on the object with a reference to a DOM node is always a potential leak, using a function to grab it when we need it avoids this risk I think.